packages/jest-config/src/normalize.js

```diff
@@ -145,23 +145,41 @@ const setupBabelJest = (options: InitialOptions) => {
 
     if (customJSPattern) {
```

Comment:
JS/TS here is so much duplicate code now that it might be worth a `[customJSPattern, customTSPattern].forEach`?